src/main/kotlin/io/snyk/plugin/ui/jcef/ThemeBasedStylingGenerator.kt

```diff
@@ -74,10 +74,7 @@ class ThemeBasedStylingGenerator {
                     val tabItemHoverColor =
                         globalScheme.getColor(ColorKey.find("INDENT_GUIDE")) // The closest color to target_rgb = RGB (235, 236, 240)
 
-                    val editorColorsManager = EditorColorsManager.getInstance()
-                    val editorColorsScheme: EditorColorsScheme = editorColorsManager.globalScheme
-                    val fontPreferences: FontPreferences = editorColorsScheme.fontPreferences
-                    val editorFont = fontPreferences.fontFamily
+                    val preBackgroundColor = if (isDarkTheme) "#000000" else "#ffffff"
```

Comment:
Why are the values hard coded? Can't we use values from the color scheme?

Reply:
https://plugins.jetbrains.com/docs/intellij/themes-customize.html#finding-attribute-keys-for-ui-controls should help to find the right entry.

Reply:
It works better now! Thank you 🙏 

<img width="640" alt="theme" src="https://github.com/user-attachments/assets/fa4f1e24-de9f-43da-bcb7-73ec94f44869">